dbfit-java/core/src/main/java/dbfit/api/AbstractDbEnvironment.java

```diff
@@ -121,19 +121,44 @@ public final PreparedStatement createStatementWithBoundFixtureSymbols(
     }
 
     @Override
+    public final StatementExecution createStatementExecutionWithBoundFixtureSymbols(TestHost th, String commandText)
+            throws SQLException {
+        return createStatementExecution(createStatementWithBoundFixtureSymbols(th, commandText));
```

Comment:
We could merge `createStatementWithBoundFixtureSymbols` into this method too I think.